src/System.Private.CoreLib/shared/System/Threading/Tasks/TaskFactory.cs

```diff
@@ -2412,7 +2414,7 @@ internal static void CommonCWAnyLogicCleanup(Task<Task> continuation)
         {
             // Force cleanup of the promise (e.g. removing continuations from each
             // constituent task), by completing the promise with any value.
-            ((CompleteOnInvokePromise)continuation).Invoke(null);
+            ((CompleteOnInvokePromise)continuation).Invoke(Task.CompletedTask);
```

Comment:
What difference will this cause? I just want to understand why passing null was wrong.

Reply:
> What difference will this cause?

Nothing at all; it's not observable.  I'll just change it back to be null and use ! in a few places.

Reply:
You don’t need to change it I was just curious.